README.md

```diff
@@ -27,5 +27,4 @@ This project has dependencies on other open-source projects. These projects are
 
 |Project|Author|Sources|License|
 |--------|-----|---|---------|
-|Fizzler|Atif Aziz (@atifaziz)|[GitHub](https://github.com/atifaziz/Fizzler)|[LGPL](https://github.com/atifaziz/Fizzler/blob/master/COPYING.txt)|
```

Comment:
is now removed from the nuget package as dependency is now only used as unit test dependency.